nginx/lua/webdav_read_header_filter.lua

```diff
@@ -0,0 +1,11 @@
+local ngx = require("ngx")
+local fileutil = require("fileutil")
+
+
+local path = fileutil.get_request_local_path()
```

Comment:
We should only look up the path if we need to fetch adler32. Non-TPC requests will not ask for it